docs/beyond_basics/101-164-dataladdening.rst

```diff
@@ -0,0 +1,141 @@
+If you're really pressed for time because your dog is sick, your toddler keeps eating your papers and your boss is behind you with a whip, the findoutmore below summarizes the most important sections from the Basics for you to read:
+
+.. findoutmore:: The Basics for the impatient
+
+   To get a general idea about DataLad, please read sections :ref:`philo` and :ref:`executive_summary` from the introduction (reading time: 15 min).
+
+   To gain a good understanding of some important parts of DataLad, please read chapter :ref:`chapter_datasets`, :ref:`chapter_run`, and :ref:`chapter_gitannex` (reading time: 60 minutes).
+
+   To become confident in using DataLad, sections :ref:`help`, :ref:`filesystem` can be very useful. Depending on your aim, :ref:`chapter_collaboration` (for collaborative workflows), :ref:`chapter_thirdparty` (for data sharing), or :ref:`chapter_yoda` (for data analysis) may contain the relevant background for you.
+
+Prior to transforming your project, **we recommend to create a copy of it**.
+We don't believe there is much that can go wrong from the software-side of things, but data is precious, so if its your first time using DataLad, better be safe than sorry.
+
+Step 1: Planning
+^^^^^^^^^^^^^^^^
+
+The first step to DataLad-ify your project is to turn it into one or several nested datasets.
+Whether you turn a project into a single dataset or several is dependent on the current size of your project and how much you expect it to grow overtime, but also on its contents.
+You can find guidance on this in paragraph below.
+
+The next step is to save dataset contents.
+You should take your time and invest thought into this, as this determines the looks and feels of your dataset, in particular the decision on which contents should be saved into :term:`Git` or :term:`git-annex`.
+The section :ref:`symlink` should give you some necessary background information, and the chapter :ref:`chapter_config` the relevant skills to configure your dataset appropriately.
+You should consider the size, file type and modification frequency of files in your decisions as well as potential plans to share a dataset with a particular third party infrastructure.
+
+Step 2: Dataset creation
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Transforming a directory into a dataset is done with :command:`datalad create --force`.
+The ``-f``/``--force`` option enforces dataset creation in non-empty directories.
+Consider :ref:`applying procedures <procedures>` with ``-c <procedure-name>`` do apply configurations that suit your use case.
+
+.. findoutmore:: What is my directory is already a Git repository?
+
+   If you want to transform a Git repository to a DataLad dataset, a :command:`datalad create -f` is the way to go, too, and completely safe.
+   Your Git history will stay intact and will not be tempered with.
+
+If you want to transform a series of nested directories into nested datasets, continue with :command:`datalad create -f` commands in all further subdirectories.
```

Comment:
It might be good to advise people to start at the top and them do a `datalad create -d^ -f` in each directory they want to become a subdataset. This way, new subdatasets are properly registered, and a single `datalad save -r` in the root and at the very end takes care of the rest.